modules/sdk-core/src/bitgo/wallet/iWallets.ts

```diff
@@ -80,9 +81,7 @@ export const GenerateLightningWalletOptionsCodec = t.strict(
     passphrase: t.string,
     enterprise: t.string,
     passcodeEncryptionCode: t.string,
-    // custodial - bitgo controls the node private key
-    // hot - client controls the node private key by running remote signer node
-    type: t.union([t.literal('custodial'), t.literal('hot')]),
+    subType: t.union([t.literal('lightningCustody'), t.literal('lightningSelfCustody')]),
```

Comment:
A bit confused, for my own clarification, are we removing `type` entirely? Or now we'll have both `type` and `subType` field?

Reply:
user is going to only input the `subType` because lighting can only be hot. When we make the request to wallet platform it will also have `type: 'hot'`

Reply:
no need for type. we are going to always use hot do client only needs to pass subType